README.md

```diff
@@ -242,11 +242,9 @@ To install from a Debian Repository:
 
      For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
    
-      	wget -q0 – http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | 
-
-      	sudo apt-key add -echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' | 
-
-      	sudo tee /etc/apt/sources.list.d/rocm.list
+      	wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key |
+        sudo sudo apt-key add -sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main 
+        > /etc/apt/sources.list.d/rocm.list'
```

Comment:
These lines should be replaced by the following 2 lines, with a line containing ``` before and after them to ensure the lines are treated as a code block:

```
wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | sudo apt-key add -
sudo sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main > /etc/apt/sources.list.d/rocm.list'
```